homepageapp/tasks.py

```diff
@@ -0,0 +1,18 @@
+from celery import shared_task
+from django.contrib.auth.models import User
+from django.utils import timezone
+import datetime
+
+
+@shared_task(bind=True)
+def delete_inactive_users_func(self):
+
+    current_time = timezone.now()
+    users = User.objects.all()
+    for user in users:
+        if user.is_active is False and current_time - datetime.timedelta(days=1) > user.date_joined:
+            user.delete()
```

Comment:
```suggestion
    yesterday =  datetime.timedelta(days=1)
    users = User.objects.filter(is_active = False, user.date_joined_lt=yesterday)
    users.delete()
```

Further remarks:
- avoid computing variables in loop if not needed - here: yesterday
- try to narrow quryset by queryset not in loop.
- loop is not needed, django will handle it in more efficient way.
- I am not 100% this queryset will work, please double check me